doc/how_to_validate.rst

```diff
@@ -0,0 +1,64 @@
+..
+  SPDX-FileCopyrightText: 2021 The PyPSA meets Earth authors
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _tutorial::
+
+##########################################
+How to validate?
+##########################################
+
+.. TODO add a list of actions needed to do the validation
+
+Data workflow is PyPSA-Earth allows to extract all the data needed to build your energy model from global open data sources. An example of the validation procedure is available in the `Nigeria validation <https://github.com/pypsa-meets-earth/documentation/blob/main/notebooks/validation/validation_nigeria.ipynb>`_ notebook. Public information on the power system of Nigeria are compared to those obtained from the PyPSA-Earth model.
+
+Simulation procedure
+===================================
+
+It may be recommended to check the following quantities the validation:
+
+#. inputs used by the model:
+
+    #. network characteristics;
+
+    #. substations;
+
+    #. installed generation by type;
+
+#. outputs of the simulation:
+
+    #. demand;
+
+    #. energy mix.
+
+Data sources
```

Comment:
"Third-party data sources for validation" or something like that? feel free to improve on it.
The message below should be: to do the validation you need to compare the data obtained with the workflow with other (third-party) trustworthy data, such as those that follow

Reply:
Tried to improve